bin/pycbc_single_template

```diff
@@ -293,8 +295,18 @@ with ctx:
     parse_row.params = t()
     for param in row.fieldnames:
         setattr(parse_row.params, param, row[param][0])
-    chisq_bins = int(vetoes.SingleDetPowerChisq.parse_option(parse_row,
-                                                               opt.chisq_bins))
+
+    try:
+        chisq_bins = int(vetoes.SingleDetPowerChisq.parse_option(parse_row,
```

Comment:
Just for extra paranoia, I'd also add a check for chisq_bins >= 2, and fall back if that fails